core/os/android/adb/commands.go

```diff
@@ -371,6 +372,10 @@ func (b *binding) QueryPerfettoGpuProfilingDataSources(ctx context.Context) (*de
 			gpu.HasRenderStage = true
 			continue
 		}
+		if desc.GetName() == gpuMemTotalDataSourceDescriptorName {
```

Comment:
update the comment of this function. We should probably add a comment to set `HasGpuMemTotal` to true always once it's enforced by CDD

Reply:
Comment for this function has been updated. As for the comment for cdd enforcement, for this kernel tracking system, GMS will most likely be the proper vehicle. Do we still want to mention this? unlike Frame Life Cycle, we would always need to back support any volunteered R devices.

Reply:
ACK